readthedocs/rtd_tests/tests/test_privacy_urls.py

```diff
@@ -337,6 +337,8 @@ def setUp(self):
             'api_webhook_generic': {'status_code': 403},
             'remoteorganization-detail': {'status_code': 404},
             'remoterepository-detail': {'status_code': 404},
+            # This isn't a valid media type
+            '/api/v2/build/1/log.json': {'status_code': 404},
```

Comment:
For some reason the test suite insists on getting this media type for the /log endpoint.

Reply:
Why `,json`? We are dealing with `.txt`, but I'd say that the extension is not necessary, right?

Reply:
Yeah, this is just for implicit sending the media type rather than do it on the http headers. 
This test https://github.com/rtfd/readthedocs.org/blob/761f14dfc72de7a5c9a4f4e874fda613d4548d8c/readthedocs/rtd_tests/tests/test_privacy_urls.py#L351 cheks for all endpoints reversing all posible urls (including media types explicity on the url), for the /log endpoint two urls are generated `.txt` and `.json` (but only `txt` media type is supported) I think is because the main media type is `json`, so I _black listed_ that url.

Reply:
Mmm... Can we avoid the generation of `.json` since we don't need this here?

Reply:
Well, I try with the recent change, but the CI fails :(. I think this is something related to the reversing process that I mention, should I investigate more deeper here? O leave it as is (since is related to the test suite, not the _real code_, I think)?